rotkehlchen/chain/evm/frontend_structures/types.py

```diff
@@ -0,0 +1,46 @@
+from enum import auto
```

Comment:
As mentioned before. Concept of `frontend_structures` is mistaken. This file should not be under this directory.